odoo/openupgrade/doc/source/modules120-130.rst

```diff
@@ -407,7 +407,7 @@ missing in the new release are marked with |del|.
 +----------------------------------------------+-------------------------------------------------+
 |lunch                                         |                                                 |
 +----------------------------------------------+-------------------------------------------------+
-|mail                                          |                                                 |
+|mail                                          | Work in progress                                |
```

Comment:
Anything else? Or can it go to Done?

Reply:
I think a better set a done state after migrating Mass mailing module.

Reply:
`mass_mailing` has nothing to do with this one and in fact most of the DBs doesn't install this "marketing tool".